context/app/static/js/shared-styles/tables/EntitiesTable/EntitiesTables.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+
+import { Tab, Tabs } from 'js/shared-styles/tabs';
+import { useSearchTotalHitsCounts } from 'js/hooks/useSearchData';
+import { entityIconMap } from 'js/shared-styles/icons/entityIconMap';
+import EntityTable from './EntityTable';
+import { EntitiesTabTypes } from './types';
+import { StyledTabPanel } from './style';
+
+interface EntitiesTablesProps<Doc> {
+  isSelectable?: boolean;
+  initialTabIndex?: number;
+  entities: EntitiesTabTypes<Doc>[];
+}
+
+const singleTabProps = {
+  'aria-disabled': true,
+  disableRipple: true,
+  sx: { cursor: 'default' },
+};
```

Comment:
Does this offer advantages over using `disabled={entities.length === 1}`? https://mui.com/material-ui/react-tabs/#disabled-tab

Reply:
Styling. The MUI disabled state is grayed out.